sc2reader/data/attributes.json

```diff
@@ -716,7 +716,8 @@
         "Tych": "Tychus",
         "Vora": "Vorazun",
         "Zaga": "Zagara",
-        "Zera": "Zeratul"
+        "Zera": "Zeratul",
+        "Meng": "Mengsk"
```

Comment:
Please keep this list in sorted order because it make it easier to spot duplicates.